README.md

```diff
@@ -25,37 +25,57 @@ These directions below assume you are using AWS.  We will include additional clo
 
 ### Collecting Information
 
-Make sure you know what region you're running in (e.g. `us-west-2`).
+The installation script will prompt for several question. It is useful prepare for some of the answers beforehand to expedite the installation process. Here are a few pointers:
 
-### Starting an AWS VM
+* make sure you know what region you're running in (e.g. `us-west-2`)
+* decide whether you want to create an instance for development or production as it might impact the size and therefore the cost of the host virtual machine
+* find out whether your favorite editor is installed on host virtual machine
+* be sure to know how to create a static IP address for your virtual machine (AWS calls this _Elastic IP_)
```

Comment:
It would be nice if you explained how to do it, or pointed to a link on how to do it.